### DISCUSSION TITLE
Renovate docker image's major version tag should be the latest version

### DISCUSSION THREAD

**Discussion by attekemppila** [renovatebot/renovate#10233] [Suggest an Idea]
I'm running self-hosted Renovate using Docker image renovate/renovate:25 in self-hosted Gitlab. Renovate is also set up to create a pr on major releases (no auto-merge).

I had assumed that renovate/renovate:25 would be an alias to the latest 25.x.y version but this is not the case at the moment. renovate/renovate:25 image uses renovate 25.5.0. Though the image is not even the same as 25.5.0 tag.

* [renovate/renovate:25](https://hub.docker.com/layers/renovate/renovate/25/images/sha256-0ef104f1ae6e396b6cb5fdc6d258c21740a43a6d5ccf63f5194489fe13a90f99)
* [renovate/renovate:25.5.0](https://hub.docker.com/layers/renovate/renovate/25.5.0/images/sha256-ccb6aa836e83e33822913f6cfdf88d01875e4b998c395a320971d75357563312)

(Is the current situation intentional, oversight or bug.)

Anyway, what I'm suggesting is that **renovate/renovate:\<major\> tag to always be the latest major.x.y release**. Or if not, wouldn't it be clearer to just remove the renovate/renovate:25 tag (or not create major version tags in the future).

Other options I could do but would prefer not to:

* Use latest - Probably breaks on major updates. That's why I prefer to use Renovate pr for major releases, as mentioned.
* Use 25.x.y tag and let Renovate to auto-merge non-major updates - I prefer not to, since it just creates unnecessary noise in the repo.

**Comment by viceice** [comment]
This is a bug, the short tags should always follow latest matching version. I'll check the builder.

**Reply by rarkins** [reply]
FYI `latest` seems correct, and it will be v25 for quite some time

**Comment by viceice** [comment]
Btw: you should checkout our official renovate runner templates for gitlab

https://gitlab.com/renovate-bot/renovate-runner

**Comment by viceice** [comment]
Fix released, need to bubble through. So should be finished today afternoon or evening.

**Reply by viceice** [reply]
https://github.com/renovatebot/internal-tools/releases/tag/v1.6.10
